packages/mongodb-access-adapter/datagrip-access-adapter/src/main/kotlin/com/mongodb/jbplugin/accessadapter/datagrip/adapter/DataGripMongoDbDriver.kt

```diff
@@ -79,6 +81,8 @@ internal class DataGripMongoDbDriver(
 
     override suspend fun connectionString(): ConnectionString = ConnectionString(dataSource.url!!)
 
+    override suspend fun <S> explain(query: Node<S>): ExplainPlan = ExplainPlan.CollectionScan
```

Comment:
I guess this is the hardcoded part?

Reply:
Yes! We are not running the explain yet so I'm just returning CollectionScan in the meantime until we have a proper implementation.